AzureCommunicationUI/AzureCommunicationUI/CallCompositeOptions/LocalizationConfiguration.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+import Foundation
+import SwiftUI
+
+public struct LocalizationConfiguration {
+    let locale: String
+    let localizableFilename: String
+    let customStrings: [String: String]
+    let isRightToLeft: Bool
+
+    /// Create an instance of LocalizationConfiguration to customize localization.
+    /// - Parameter locale: Name of the language locale.
```

Comment:
I worry "Name" might be a little confusing. Developers might accidentally put in the "french" as a string. And since it is just a string there is a lot of room of interpretation and confusion
```suggestion
    /// - Parameter locale: String representing the locale code (ie. en, en_CA, fr, fr_FR)
```

Reply:
The comment is not finalized, still need to sync with Android, will have a separate PR to correct them. Good suggestion locale code is probably better =)